modules/tinymce/src/plugins/fullscreen/main/ts/core/Actions.ts

```diff
@@ -78,22 +79,32 @@ const viewportUpdate = VisualViewport.get().fold(
   }
 );
 
-const toggleFullscreen = function (editor, fullscreenState) {
+const toggleFullscreen = (editor: Editor, fullscreenState: Cell<any>) => {
   const body = document.body;
   const documentElement = document.documentElement;
-  let editorContainerStyle;
-  let editorContainer, iframe, iframeStyle;
-  editorContainer = editor.getContainer();
+  const editorContainer = editor.getContainer();
   const editorContainerS = Element.fromDom(editorContainer);
 
   const fullscreenInfo = fullscreenState.get();
   const editorBody = Element.fromDom(editor.getBody());
 
   const isTouch = Env.deviceType.isTouch();
 
-  editorContainerStyle = editorContainer.style;
-  iframe = editor.getContentAreaContainer().firstChild;
-  iframeStyle = iframe.style;
+  const editorContainerStyle = editorContainer.style;
+  const iframe = editor.getContentAreaContainer().firstChild as any;
```

Comment:
I know this wasn't something you did, but this should probably be `editor.iframeElement`?

Reply:
So that's what those are? I tried a few of our different Element and Nodes, but didn't get something that worked. I'll keep that in mind for the future.